paper/paper.md

```diff
@@ -0,0 +1,174 @@
+date: 29 October 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+With the development of the Brain Imaging Data Structure
+(``BIDS``; [@Gorgolewski2016]), the neuroscientific community received
+a standard to organize and share data. BIDS standardizes storage of complex experimental
+data by prescribing a set of file naming conventions and templates to store the
+associated metadata in the form of Javascript Object Notation (JSON) files. It thus
+facilitates data sharing, eases metadata querying and enables automatic data
+analysis pipelines.
+
+While BIDS was originally intended for Magnetic Resonance Imaging (MRI)
+data, it has recently been extended for electro-/magneto-encephalography (EEG/MEG)
+[@Niso2018, @Pernet2019], and intracranial EEG (iEEG) [@Holdgraf2019].
+In this context, many software packages for electrophysiology data analysis are
+now becoming BIDS compatible. BIDS is a rich system to curate, aggregate and
+annotate neuroimaging databases. For this reason we developed a dedicated
+software, ``MNE-BIDS`` with the goal to provide
+a programmable interface for BIDS datasets in electrophysiology. ``MNE-BIDS`` allows
+users to re-organize data into BIDS formats, store associated
+metadata after anonymization, extract information necessary for preprocessing and
+read them into ``MNE-Python`` objects [@Agramfort2013], ready for source localization.
+
+Starting with a single directory full of data files with arbitrary
+names, ``MNE-BIDS`` can be used to extract existing metadata, reorganize the
+files into the BIDS format, and write additional metadata.
+All the conversion routines are thoroughly tested by running the output through the
+[BIDS validator](https://github.com/bids-standard/bids-validator).
+Moreover, ``MNE-BIDS`` supports converting data formats that are not BIDS
+compatible into permissible formats. As a result of this, users can easily convert
+their datasets to BIDS in a matter of minutes, rather than hours of manual
+labour.
+
+In addition to this core functionality, ``MNE-BIDS`` is continuously being
+extended to facilitate the analysis of BIDS formatted data.
+To name some features, it is possible to read a BIDS dataset and obtain a
+Python object, ready for analyis with ``MNE-Python``.
+Users can save a T1-weighted anatomical MRI image alongside the MEG or EEG data
+and apply an automatic defacing algorithm for anonymization purposes.
+As a last example, anatomical landmark coordinates can be saved to enable coregistration
+between the MEG/EEG and MRI data, which is necessary for
+computation of forward and inverse solutions.
+
+Users can easily install ``MNE-BIDS`` on all major platforms via `pip` and
+`conda`, and its functionality is continuously tested on Windows, OS X, and
+Linux.
+Other than the core dependencies for scientific computing
+(`numpy`, `scipy`) and handling of MEG/EEG data (`mne`),
+``MNE-BIDS`` has minimal dependencies, all of which are optional. The Application
```

Comment:
what do you have in mind for optional deps? I see only nibabel. If would rather be explicit that it needs numpy, scipy, mne and optionally nibabel.

Reply:
agreed!

Reply:
I think he might have also meant `pybv` as soon as that lands

Reply:
and then also `nilearn` for `from nilearn.plotting import plot_anat`, which we use in an example to show how plotting coregistered fiducials works.

And `scikit-learn`, because it is a dependency of `nilearn` :-)

I'll take a proposal if someone has a nice way to put this in a sentence.